docs/source/developer.rst

```diff
@@ -131,3 +131,75 @@ Documenting Changes
 
 When you add a public method, please document it in the usage docs and the test
 mode docs. Follow the format for already-existing methods.
+
+Uploading to PyPi
+-----------------
+
+When new features are added or bugs are fixed it is necessary to push Pymesync
+to `PyPi`_ so users can ``pip install`` those changes. This can only be done by
+owners and maintainers of Pymesync, listed below. Email support@osuosl.org if
+you believe a new version is warranted.
+
+====================== ==========
+Developer              Role
+====================== ==========
+Matthew Johnson (mrsj) Owner
```

Comment:
You should note that the names (in parenthesis) are nicks on irc.freenode.net or  include GH usernames.